src/app/tabs/run-tab.js

```diff
@@ -581,6 +584,53 @@ function settings (container, self) {
       }
     })
   }
+  function signMessage (event) {
+    self._deps.udapp.getAccounts((err, accounts) => {
+      if (err) { addTooltip(`Cannot get account list: ${err}`) }
+      var stringifyRSV = (rsv) => {
+        var rsvJson = () => { return JSON.stringify(rsv, null, '\t') } 
+        copyToClipboard(rsvJson).onclick(event)
```

Comment:
The copy to clipboard seems not working when using `web3` ...

Reply:
I was connected to a local geth using web3 provider

Reply:
@yann300 Hmm I was able to get it working using local geth with web3 provider...I will provide a VM in a few minutes...
did you run ./geth -syncmode "light" --rpc --rpcapi eth,web3,personal --rpccorsdomain "*"?

Reply:
it was working but the content was not copied to the clipboard...

Reply:
@yann300 what browser/version are you on

Reply:
chromium 62.0.3202.62 
copying a "vm" signed message worked, but not a msg signed using web3...
let me retry though

Reply:
got the same, that is very weird... are you able to reproduce?

Reply:
@yann300 hmmm no error with chrome and web3 and geth on my end. Can you forward anything red you see in the Developer Tools console in Chrome?

Reply:
i don't have any error with web3 but still the coded msg is not copied to the clipboard..

Reply:
@yann300 could you try now

Reply:
same .... the copy function works when the javascript VM is used, but not with web3...
I also tried to directly use `const copy = require('clipboard-copy')` ...
that's really weird

Reply:
Could you replicate ? (that 's also not working on firefox)

Why not removing the auto copy to clipboard and to display r,s,v (as hex string) and the raw signed message in a pre HTML tag. then user would need to manually copy it.
I fear there might be issue with this copytoclipboard lib in the context we are using it...

Reply:
@yann300 you are correct! I was able to replicate but I fixed it by avoiding the clipboard libraries with these lines. Could you pull the latest commit and try testing that one?
```document.execCommand('copy')
            if (navigator.clipboard) {
              navigator.clipboard.writeText(rsv)
            }```

Reply:
@yann300 Pre HTML tag will fix the issue, and I think the context would actually fit the copyToClipboard library (in Firefox), and allow it to work. I do see the message : 
```
```
    document.execCommand(‘cut’/‘copy’) was denied because it was not called from inside a short running user-generated event handler.
```
So this would circumvent that issue

Reply:
@yann300 Could you test this change in the newest commit